### ISSUE TITLE
[SNOWFLAKE] Ingestion logs ERROR message for not parsing VARIANT Datatype

### ISSUE THREAD

**Issue by akash-jain-10** [open-metadata/OpenMetadata#2417]
```
[2022-01-25 15:53:37,860] ERROR    {metadata.ingestion.source.sql_source:561} - 'VARIANT' object has no attribute 'length' : {'name': 'related_same_month_brand', 'type': VARIANT(), 'nullable': True, 'default': None, 'autoincrement': False, 'comment': None, 'primary_key': False}
Traceback (most recent call last):
  File "OpenMetadata/venv/lib/python3.9/site-packages/metadata/ingestion/source/sql_source.py", line 506, in _get_columns
    col_data_length = self._check_col_length(
  File "OpenMetadata/venv/lib/python3.9/site-packages/metadata/ingestion/source/sql_source.py", line 576, in _check_col_length
    return col_raw_type.length if col_raw_type.length else 1
```

cc - @ayush-shah , @harshach

**Comment by ayush-shah** [comment]
@codingwithabhi can you pick this issue up as well?

**Comment by codingwithabhi** [comment]
I will work on this

**Comment by harshach** [comment]
@codingwithabhi @ayush-shah any updates?

**Comment by ayush-shah** [comment]
fixed with #2622
